### PR TITLE
No transforms

### PR DESCRIPTION
**Pull request by eggrobin**

### CODE REVIEW

File: ksp_plugin/interface.hpp [hunk 1]
```diff
@@ -6,12 +6,9 @@
 #include "base/pull_serializer.hpp"
 #include "base/push_deserializer.hpp"
 #include "ksp_plugin/plugin.hpp"
-#include "physics/transforms.hpp"
 
 namespace principia {
```
Comment (pleroy) [inline ksp_plugin/interface.hpp]: Remove empty line.

File: ksp_plugin/plugin.cpp [hunk 2]
```diff
@@ -492,23 +492,18 @@ FrameField<World> Plugin::Navball(
...
-  DiscreteTrajectory<Barycentric> apparent_trajectory;
-  apparent_trajectory.Append(intermediate_it.time(),
-                             intermediate_it.degrees_of_freedom());
-  return Normalize(
-    Identity<WorldSun, World>()(
-        BarycentricToWorldSun()(
-            apparent_trajectory.last().degrees_of_freedom().velocity())));
+  Instant const& time = vessel.prolongation().last().time();
+  DegreesOfFreedom<Barycentric> const& degrees_of_freedom =
+      vessel.prolongation().last().degrees_of_freedom();
```
Comment (pleroy) [inline ksp_plugin/plugin.cpp]: Evaluate `last` only once, it's not free.

File: ksp_plugin/plugin.cpp [hunk 3]
```diff
@@ -492,23 +492,18 @@ FrameField<World> Plugin::Navball(
...
+  return Identity<WorldSun, World>()(
+      BarycentricToWorldSun()(
+          rendering_frame->FromThisFrameAtTime(time).orthogonal_map()(
+              rendering_frame->FrenetFrame(
+                  time,
+                  rendering_frame->ToThisFrameAtTime(time)(degrees_of_freedom))(
+                      Vector<double, Frenet<Rendering>>({1, 0, 0})))));
```
Comment (pleroy) [inline ksp_plugin/plugin.cpp]: Maybe a comment explaining what this does?

File: ksp_plugin/plugin.hpp [hunk 4]
```diff
@@ -20,6 +20,7 @@
 #include "physics/body.hpp"
 #include "physics/discrete_trajectory.hpp"
 #include "physics/ephemeris.hpp"
+#include "physics/dynamic_frame.hpp"
```
Comment (pleroy) [inline ksp_plugin/plugin.hpp]: Alphabet, how does it work?

File: ksp_plugin/plugin.hpp [hunk 5]
```diff
@@ -207,7 +207,7 @@ class Plugin {
   // relation between |WorldSun| and |World|.  No transfer of ownership.
   virtual RenderedTrajectory<World> RenderedVesselTrajectory(
       GUID const& vessel_guid,
-      not_null<RenderingTransforms*> const transforms,
+      not_null<RenderingFrame*> const transforms,
```
Comment (pleroy) [inline ksp_plugin/plugin.hpp]: `transforms`?

File: ksp_plugin_test/plugin_integration_test.cpp [hunk 6]
```diff
@@ -588,7 +589,8 @@ TEST_F(PluginIntegrationTest, Prediction) {
   plugin.InsertSun(celestial, SIUnit<GravitationalParameter>());
   plugin.EndInitialization();
   EXPECT_TRUE(plugin.InsertOrKeepVessel(satellite, celestial));
-  auto transforms = plugin.NewBodyCentredNonRotatingTransforms(celestial);
+  auto rendering_frame =
```
Comment (pleroy) [inline ksp_plugin_test/plugin_integration_test.cpp]: const?

File: ksp_plugin/interface.hpp (hunk 1 above)
Comment (eggrobin) [inline ksp_plugin/interface.hpp]: Done.

File: ksp_plugin/plugin.cpp (hunk 2 above)
Comment (eggrobin) [inline ksp_plugin/plugin.cpp]: Done.

File: ksp_plugin/plugin.cpp (hunk 3 above)
Comment (eggrobin) [inline ksp_plugin/plugin.cpp]: Done.

**Pull request merged**